### PR TITLE
Mar 8, 2023

### PR DESCRIPTION
**Pull request by nickdesaulniers**
Mar 8, 2023

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ invite and meet link.
 
 Send to: llvm@lists.linux.dev
 
+## Mar 8, 2023
+- (Nick) scaling down CI
+- (Nick/Bill) ["p" constraint](https://lore.kernel.org/lkml/20230303231133.1486085-1-eranian@google.com/)
+- (Connor) [riscv breakages due to extensions moving out of I spec](https://github.com/ClangBuiltLinux/linux/issues/1808)
```
Comment (ConchuOD) [inline README.md:12]: Not worth fixing, but my name has 1 n :)

File: README.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ invite and meet link.
...
+- [riscv 5.10 build is broken when using BFD
```
Comment (nathanchance) [inline README.md:13]: These are the same issue

File: README.md [hunk 3]
```diff
@@ -6,6 +6,17 @@ invite and meet link.
...
+- [ICC support is gone](https://www.phoronix.com/news/Linux-6.3-Drops-Intel-ICC)
+- (Nathan) [ppc mtune flags](https://reviews.llvm.org/D144967)
+- ppc64le elfv2 testing possible now
```
Comment (nathanchance) [inline README.md:16]: big endian, not little endian

**Review by nathanchance — APPROVED** [review]
LGTM aside from a few nits

File: README.md (hunk 2 above)
Comment (ConchuOD) [inline README.md:13]: Was this line supposed to have a ](<link>)?

Comment (nickdesaulniers) [reply README.md:13]: oh, I thought I had deleted this line when I realized that. I must hit ctrl+z at some point!

**Pull request merged**